pkg/store/bucket.go

```diff
@@ -276,6 +282,11 @@ func newBucketStoreMetrics(reg prometheus.Registerer) *bucketStoreMetrics {
 		Help: "Total number of empty postings when fetching block series.",
 	})
 
+	m.emptyStreamResponses = promauto.With(reg).NewCounter(prometheus.CounterOpts{
+		Name: "thanos_bucket_store_empty_blocks_total",
```

Comment:
The name is a little bit confusing. The block itself may not be empty, just the response from `Recv` could be empty due to various reasons. What do you think about renaming the metric?

Reply:
This metric is used by the store proxy to count how many stores return no response for a given select. I wonder if this metric makes sense for blocks because we have counter for empty postings anyway. Should we remove this metric and inject a dummy counter instead?

Reply:
Yeah dummy counter is better IMO.

Reply:
Looks like the empty postings metric counts the same thing, so I injected that one instead.